src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/UtentiPGPage.java

```diff
@@ -0,0 +1,349 @@
+            By surnameField = By.xpath("//p[contains(text(),'Cognome')]");
+            By codiceFiscaleField = By.xpath("//p[contains(text(),'Codice Fiscale')]");
+            By emailField = By.xpath("//p[contains(text(),'Email')]");
+
+
+            getWebDriverWait(10).withMessage("il bottone indietro della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(indietroButton));
+            getWebDriverWait(10).withMessage("il titolo della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(titolo));
+            getWebDriverWait(10).withMessage("il bottone modifica della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(modificaButton));
+            getWebDriverWait(10).withMessage("il bottone rimuovi della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(rimuoviButton));
+            getWebDriverWait(10).withMessage("la sezione nome della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(nameField));
+            getWebDriverWait(10).withMessage("la sezione cognome della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(surnameField));
+            getWebDriverWait(10).withMessage("la sezione codice fiscale della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(codiceFiscaleField));
+            getWebDriverWait(10).withMessage("la sezione email della pagina riepilogativa non è visibile").until(ExpectedConditions.visibilityOfElementLocated(emailField));
+            logger.info("Si visualizza correttamente pagina riepilogativa");
+        }catch (TimeoutException e){
+            logger.error("Non si visualizza correttamente pagina riepilogativa con errore:"+e.getMessage());
+            Assert.fail("Non si visualizza correttamente pagina riepilogativa con errore:"+e.getMessage());
+        }
+    }
+
+    public void clickModifyButton(){
+        getWebDriverWait(30).withMessage("il bottone modifica non è cliccabile").until(ExpectedConditions.elementToBeClickable(modificaButton));
+        logger.info("Si clicca sul bottone modifica");
+        modificaButton.click();
+    }
+
+    public void checkEmailBoxActive(){
+      String checkEmail = emailBox.getAttribute("disabled");
+      String checkConfirmEmail =  confirmEmailBox.getAttribute("disabled");
+
+       if(checkEmail==null && checkConfirmEmail==null){
+           logger.info("il campo email e il campo conferma email sono attivi");
+       }else {
+           logger.error("il campo email e il campo conferma email sono disattivi");
+           Assert.fail("il campo email e il campo conferma email sono disattivi");
+       }
+    }
+
+    public void inserisciNuovoEmail(String newMail){
+        emailBox.sendKeys(Keys.chord(Keys.CONTROL,"a"),Keys.DELETE);
```

Comment:
what's the purpose of this action?

Reply:
purpose of this action is to clear the textbox field, as emailBox.Clean() doesnt work for this case